search/lib/api/provider.js

```diff
@@ -9,6 +9,7 @@ async function getProvider (request, response) {
   try {
     const providerId = request.params.providerId;
     logger.info(`GET /${providerId}`);
+    const pageSize = request.query.limit || 10;
```

Comment:
Should move the `Number` conversion here so the `const pageSize` is always a number type